code/modules/supermatter/supermatter.dm

```diff
@@ -574,6 +607,30 @@ var/list/supermatter_final_thoughts = list(
 /obj/machinery/power/supermatter/get_artifact_scan_data()
 	return "Superdense crystalline structure - appears to have been shaped or hewn, lattice is approximately 20 times denser than should be possible."
 
+/obj/machinery/power/supermatter/proc/update_sound()
+	if(!sound_id)
+		sound_id = "[type]_[sequential_id(/obj/machinery/power/supermatter)]"
+
+	switch(damage)
+		if(0 to warning_point)
+			working_sound = 'sound/machines/supermatter/sm_calm.ogg'
+		if(warning_point to INFINITY)
+			working_sound = 'sound/machines/supermatter/sm_delam.ogg'
+	if(power > 0)
+		var/volume = 10 + max(100, (15*power))
+		if(!sound_token)
```

Comment:
Bit confused here. You only seem to null out the sound token if `power` is <= 0, rather than if it's <= `warning_point`. Doesn't that mean it will never progress to the delamination background sound unless it goes from 0 to delaminating between Process() ticks?